modules/ducktests/tests/ignitetest/tests/suites/slow_suite.yml

```diff
@@ -15,3 +15,6 @@
 
 discovery:
   - ../discovery_test.py
+
+snapshot:
+  - ../snapshot_test.py
```

Comment:
why this suite is slow? how about to make it fast?

Reply:
index_validate and idle_verify take a long time

`01:03:33  test_id:    ignitetest.tests.snapshot_test.SnapshotTest.snapshot_test.ignite_version=2.9.0
01:03:33  status:     PASS
01:03:33  run time:   4 minutes 56.490 seconds
01:03:33  test_id:    ignitetest.tests.snapshot_test.SnapshotTest.snapshot_test.ignite_version=dev
01:03:33  status:     PASS
01:03:33  run time:   23 minutes 57.322 seconds`

Reply:
Do we really need so long checks?
Why it takes so long? Do we check snapshots properly?

Why Dev is so slow?
@Mmuzaf Could you please check this? 
Seems we have Dev 5 times slower than 2.9.0

Reply:
long execution due to checking the indexes:

[DEBUG - 2021-01-22 14:41:10,996 - remoteaccount - _log - lineno:160]: ducker@tkles-pprb00277.vm.esrt.cloud.sbrf.ru: Running ssh command: /opt/ignite-dev/bin/control.sh --host 10.53.211.13 --cache validate_indexes
[DEBUG - 2021-01-22 14:53:49,009 - control_utility - __run - lineno:331]: Output of command --cache validate_indexes on node tkles-pprb00277.vm.esrt.cloud.sbrf.ru, exited with code 0, is Control utility [ver. 2.10.0-SNAPSHOT#20210122-sha1:DEV]
2021 Copyright(C) Apache Software Foundation
User: ducker
Time: 2021-01-22T14:41:12.376
Command [CACHE] started
Arguments: --host 10.53.211.13 --cache validate_indexes 
--------------------------------------------------------------------------------
no issues found.

Command [CACHE] finished with code: 0
Control utility has completed execution at: 2021-01-22T14:53:48.986
Execution time: 756610 ms

Reply:
fixed the classpath
`
================================================================================
02:06:30  test_id:    ignitetest.tests.snapshot_test.SnapshotTest.snapshot_test.ignite_version=2.9.0
02:06:30  status:     PASS
02:06:30  run time:   5 minutes 17.370 seconds
02:06:30  --------------------------------------------------------------------------------
02:06:30  test_id:    ignitetest.tests.snapshot_test.SnapshotTest.snapshot_test.ignite_version=dev
02:06:30  status:     PASS
02:06:30  run time:   5 minutes 23.372 seconds
--------------------------------------------------------------------------------
`